frontend/javascripts/oxalis/view/settings/export_bounding_box_modal.js

```diff
@@ -2,54 +2,82 @@
 import { Button, Modal, Alert } from "antd";
 import React, { useState } from "react";
 import type { BoundingBoxType } from "oxalis/constants";
-import type { APIDataset } from "types/api_flow_types";
+import type { VolumeTracing } from "oxalis/store";
+import type { APIDataset, APIDataLayer } from "types/api_flow_types";
 import { startTiffExportJob } from "admin/admin_rest_api";
+import Model from "oxalis/model";
 import features from "features";
 import * as Utils from "libs/utils";
 
 type Props = {
   destroy: () => void,
+  volumeTracing: ?VolumeTracing,
   dataset: APIDataset,
-  hasVolumeTracing: boolean,
   boundingBox: BoundingBoxType,
 };
 
-const ExportBoundingBoxModal = ({ destroy, dataset, hasVolumeTracing, boundingBox }: Props) => {
+const ExportBoundingBoxModal = ({ destroy, dataset, boundingBox, volumeTracing }: Props) => {
   const [startedExports, setStartedExports] = useState([]);
 
   const handleClose = () => {
     destroy();
   };
 
-  const handleStartExport = layerName => {
-    startTiffExportJob(
+  const exportKey = layerInfos => (layerInfos.layerName || "") + (layerInfos.tracingId || "");
+
+  const handleStartExport = async layerInfos => {
+    setStartedExports(startedExports.concat(exportKey(layerInfos)));
+    if (layerInfos.tracingId) {
+      await Model.ensureSavedState();
+    }
+    await startTiffExportJob(
       dataset.name,
       dataset.owningOrganization,
-      layerName,
       Utils.computeArrayFromBoundingBox(boundingBox),
+      layerInfos.layerName,
+      layerInfos.tracingId,
     );
-    setStartedExports(startedExports.concat(layerName));
   };
 
-  const layerNames = dataset.dataSource.dataLayers.map(layer => {
-    const nameIfFromDataset = layer.category === "color" || !hasVolumeTracing ? layer.name : null;
-    const nameIfVolume =
-      hasVolumeTracing && layer.category === "segmentation" && layer.fallbackLayerInfo != null
-        ? layer.fallbackLayerInfo.name
-        : null;
-    return nameIfFromDataset || nameIfVolume;
+  const hasMag1 = (layer: APIDataLayer) => layer.resolutions.map(r => Math.max(...r)).includes(1);
```

Comment:
you can use the ResolutionInfo abstraction like this:

```suggestion
  const hasMag1 = (layer: APIDataLayer) => getResolutionInfo(layer.resolutions).hasIndex(0);
```

`getResolutionInfo` needs to be imported from `oxalis/model/accessors/dataset_accessor`

Reply:
Done :) Though it reminds me that we should really tackle #4762 and #4893 (probably we should do this together as it is so interlinked in front-end and back-end)